tsconfig.json

```diff
@@ -14,23 +14,26 @@
     "isolatedModules": true,
     "jsx": "preserve",
     "incremental": true,
-    "types": ["react", "node"],
+    "types": ["react", "jest", "node"],
```

Comment:
Why have you added this back?

Reply:
conflict resolution error! There's been a mix-up.